stylelint-polaris/index.js

```diff
@@ -4,414 +4,503 @@ const {
   tokens,
 } = require('@shopify/polaris-tokens');
 
-/** @type {import('./plugins/coverage').PrimaryOptions} */
+/**
+ * @type {import('./plugins/coverage').CategorizedRules} The stylelint-polaris/coverage rule expects a 3-dimensional rule config that groups Stylelint rules by coverage categories. It reports problems with dynamic rule names by appending the category to the coverage plugin's rule name
+
+(e.g., Unexpected named color "blue" - Please use a Polaris color token Stylelint(stylelint-polaris/coverage/colors/color-named)")
+*/
 const stylelintPolarisCoverageOptions = {
-  colors: {
-    'color-named': 'never',
-    'color-no-hex': true,
-    'scss/function-color-relative': true,
-    'declaration-property-value-disallowed-list': {
-      opacity: [/(?!0|1)\d$|^\d{2,}|^[1-9]+\.|^\d+\.\d+\.|^0\.\d{3,}/],
-    },
-    'function-disallowed-list': [
-      // Include Sass namespace
-      // https://regex101.com/r/UdW0oV/1
-      'brightness',
-      'contrast',
-      'hue-rotate',
-      'hsl',
-      'hsla',
-      'invert',
-      'rgb',
-      'rgba',
-      'sepia',
-      /([\w-]+\.)?color-multiply/,
-      /([\w-]+\.)?color/,
-      /([\w-]+\.)?filter/,
-    ],
-    'polaris/at-rule-disallowed-list': {
-      include: [
-        // Legacy mixins
-        /([\w-]+\.)?color-icon($|\()/,
-        /([\w-]+\.)?recolor-icon($|\()/,
-        /([\w-]+\.)?control-backdrop($|\()/,
-        /([\w-]+\.)?ms-high-contrast-color/,
+  colors: [
+    {
+      'color-named': 'never',
+      'color-no-hex': [true, {message: 'Please use a Polaris color token'}],
```

Comment:
```suggestion
      'color-no-hex': true,
```
I'm guessing this was included for testing.